.kokoro/server/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net"
+
+	pb "github.com/jadekler/dummy-benchmarking-repo/.kokoro/grpc"
+	"github.com/jadekler/dummy-benchmarking-repo/storage"
+	"google.golang.org/grpc"
+)
+
+const (
+	port = ":50051"
+)
+
+type server struct{}
+
+func (s *server) Read(ctx context.Context, in *pb.ObjectRead) (*pb.Completed, error) {
+	storage.Read("a", in.BucketName, in.ObjectName)
+	fmt.Printf("%s\n", "Reading")
```

Comment:
remove (log statements are very useful for local development, but clutter up deployment environments)